Runtime/GameLiftCoreApi.cs

```diff
@@ -24,20 +25,21 @@ public class GameLiftCoreApi
     {
         private readonly GameLiftConfiguration _configuration;
         private readonly bool _isAnywhereMode;
-        private CoreApi _coreApi;
         private readonly IGameServerAdapter _gameServerAdapter;
+        private readonly CoreApi _coreApi;
 
         protected GameLiftCoreApi(GameLiftConfiguration configuration)
         {
             _configuration = configuration;
-            _coreApi = CoreApi.SharedInstance;
-            var fleetId = _coreApi.GetSetting("FleetId").Value;
-            var fleetLocation = _coreApi.GetSetting("FleetLocation").Value;
+            _coreApi = new CoreApi(Paths.PluginConfigurationFile);
```

Comment:
During a discussion with James and Shashank, there was request to use a new file as the original file was designated for Client based settings and they wanted the server information saved separately.

Reply:
That's fair, and I agree with that direction; however, my guess is there are multiple places we need to pass dependencies instead of grab shared instance or else the settings won't be in-sync as the code executes